pyhoma/models.py

```diff
@@ -153,6 +153,15 @@ def __contains__(self, name: str) -> bool:
     def __getitem__(self, name: str) -> Optional[State]:
         return next((state for state in self._states if state.name == name), None)
 
+    def __setitem__(self, name: str, state: State) -> None:
+        if self.__contains__(state.name):
+            return
+        self._states.append(
+            State(
+                **{s: getattr(state, s) for s in state.__slots__ if hasattr(state, s)}
```

Comment:
I dont get it, why recreate a new state object instead of appending the given one?

Reply:
![image](https://user-images.githubusercontent.com/5701372/89180534-3e933e00-d592-11ea-9ba5-d03e29986727.png)